mob-demo

dashabondarenko07 opened the pull request:
-контейнер не был подключен поэтому не знаю как точно будет с ним
-общий стиль button не подключен
-у меня произошел глюк на компе из-за чего папкa image  не загрузилaсь но есть в vscode

dashabondarenko07 committed:
mob-demo

-контейнер не был подключен поэтому не знаю как точно будет с ним
-общий стиль button не подключен
-у меня произошел глюк на компе из-за чего папкa image  не загрузилaсь но есть в vscode

lordponchik commented on src/partials/about.html:
```diff
@@ -0,0 +1,24 @@
+<section class="about-section">
```
section class="about-section" меняем на section class="about"

lordponchik commented on src/partials/about.html:
```diff
@@ -0,0 +1,24 @@
+<section class="about-section">
+  <div class="container">
+    <p class="about__title__label">tradition and love</p>
```
about__title__label меняем на about__label

lordponchik commented on src/partials/about.html:
```diff
@@ -0,0 +1,24 @@
+    <div class="about__cow"></div>
+    <!-- <img class="about__cow" src="../images/about/mob/about-cow.png" alt=""> -->
+
+    <p class="about__text-1">
```
about__text-1 меняем на about__text

dashabondarenko07 replied:
там цвет текста разный
добавить модификатор ?

lordponchik replied:
.about__text-1 на .about__text общие стили со 2 соединить. Если отличаются через псевдоклассы:
about__text:first-child about__text:last-child

lordponchik commented on src/partials/about.html:
```diff
@@ -0,0 +1,24 @@
+      Fusce ut velit laoreet, tempus arcu eu, molestie tortor. Nam vel justo cursus, faucibus lorem
+      eget, egestas eros. Maecenas eleifend erat at justo fringilla.
+    </p>
+    <p class="about__text-2">
```
about__text-2 на about__text

lordponchik commented on src/partials/about.html:
```diff
@@ -0,0 +1,24 @@
+      Cras massa nibh, tincidunt. Aliquam erat volutpat. Aenean accumsan.
+    </p>
+    <button class="about__btn">Read more
+        <svg class="icon-about__btn" width="5" height="8">
```
about__icon-btn

lordponchik commented on src/sass/layout/_about.scss:
```diff
@@ -0,0 +1,68 @@
+.about-section {
```
.about-section на .about

lordponchik commented on src/sass/layout/_about.scss:
```diff
@@ -0,0 +1,68 @@
+  padding-bottom: 39px;
+  background: $bg-color;
+}
+.about__title__label {
```
.about__title__label на .about__label

lordponchik commented on src/sass/layout/_about.scss:
```diff
@@ -0,0 +1,68 @@
+
+  background-color: #ffa5ba;
+}
+.about__text-1 {
```
.about__text-1 на .about__text общие стили со 2 соединить. Если отличаются через псевдоклассы:
about__text:first-child about__text:last-child

lordponchik commented on src/sass/layout/_about.scss:
```diff
@@ -0,0 +1,68 @@
+  font-size: 12px;
+  line-height: 1.6;
+}
+.about__text-2 {
```
.about__text-2  на .about__text

lordponchik commented on src/sass/layout/_about.scss:
```diff
@@ -0,0 +1,68 @@
+  margin-left: 50px;
+  border: 0;
+}
+.icon-about__btn {
```
.icon-about__btn на .about__icon-btn

dashabondarenko07 committed:
mobile-demo-2

-изменены классы
-есть вопросы к молоку (фон) ибо расположение делала только для 320 и 480 а между этими как-то там само веселится

dashabondarenko07 committed:
about-mob-fin1
